lib/Weathermap/Integrations/Cacti/WeatherMapCactiUserPlugin.php

```diff
@@ -256,9 +256,11 @@ public function handleDefaultView($request, $appObject)
         // get the locale from host app
         $locale = $this->manager->application->getLocale();
 
-        print "<h3>This is the React UI below here</h3>";
+        print "<small>This is the React UI below here</small>";
         print "<h1>INCOMPLETE</h1>";
-        print '<style src="cacti-resources/user/main.css"></style>';
+        print '<style>@import "cacti-resources/user/main.css"</style>';
+        print '<script type="text/javascript" src="overlib.js"></script>';
```

Comment:
`        print '<script type="text/javascript" src="' . $config['url_path'] . 'overlib.js"></script>';`

Reply:
Should be done now.

Reply:
In the cacti world we normally prefix with the $config['url_path'] but if you are using a relative path instead, the first `<style>` is better for efficiency.  Otherwise, the browser has to do multiple steps of retrieving the CSS, importing it into a new style block, parsing that.

Reply:
The first style might be more efficient. However, there is a (timing?) issue when loading the css.
If we do a 'src="..."' then the CSS does not load (on time), or is not applied. 
When we do a '@import ...' then the CSS is applied.

Seems strange, however this fixes some styling issues.

Reply:
That was what we experienced too. 

The '@import' fixed (works around?) this issue. At least the CSS is now always loaded.

Reply:
Interesting to know. It should use the current location of the first script.  So, the import shouldn’t make a difference.  Guess if it works, go with it as I doubt any of us have that much free time to waste when there’s a workaround 👍